From a0cda2c37c6358956ad737e227ae9bd26cdf3d35 Mon Sep 17 00:00:00 2001
From: Thiago Kenji Okada <thiagokokada@gmail.com>
Date: Tue, 16 Aug 2022 17:21:16 +0100
Subject: [PATCH 2/2] Fix paths

---
 modules/app/rss/config.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/app/rss/config.el b/modules/app/rss/config.el
index a292f4006..fe3dbfa29 100644
--- a/modules/app/rss/config.el
+++ b/modules/app/rss/config.el
@@ -20,8 +20,8 @@ easier to scroll through.")
 (use-package! elfeed
   :commands elfeed
   :init
-  (setq elfeed-db-directory (concat doom-local-dir "elfeed/db/")
-        elfeed-enclosure-default-dir (concat doom-local-dir "elfeed/enclosures/"))
+  (setq elfeed-db-directory (concat doom-cache-dir "elfeed/db/")
+        elfeed-enclosure-default-dir (concat doom-cache-dir "elfeed/enclosures/"))
   :config
   (setq elfeed-search-filter "@2-week-ago "
         elfeed-show-entry-switch #'pop-to-buffer
-- 
2.37.1

